tests/secrets/git_history/test_utils.py

```diff
@@ -392,6 +392,14 @@ def mock_git_repo_commits_too_much(self, root_folder: str, last_commit_sha: str)
     return res
 
 
+def mock_run_forever(self, root_folder: str, last_commit_sha: str) -> Dict[str, Dict[str, str]]:
+    res: Dict[str, Dict[str, str]] = {}
```

Comment:
maybe better do it as `res: List[Commit] = []` , and return also `List[Commit]`

Reply:
agreed 🙂

Reply:
will be fixed when Maryna will merge her PR :)